sdks/go/pkg/beam/transforms/xlang/python/external.go

```diff
@@ -27,7 +27,8 @@ import (
 )
 
 const (
-	pythonCallableUrn = "beam:logical_type:python_callable:v1"
+	pythonCallableUrn      = "beam:logical_type:python_callable:v1"
+	ExpansionServiceModule = "apache_beam.runners.portability.expansion_service_main"
```

Comment:
This default expansion service always has Dataframe included in it?

Reply:
yeah, we do that installation here:https://github.com/apache/beam/blob/bd8110c765d94f4640a59a4caaf1f1824e6cb889/sdks/go/pkg/beam/core/runtime/xlangx/expansionx/download.go#L409

Reply:
Ack. Sounds good for now. We probably want to make this something configurable, like we have for additional java jars for the Java expansion service.

Reply:
yes, for runinference transform, we will use the `extraPackages` argument because it needs sklearn and other additional packages.